portfolio/src/main/webapp/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Home</title>
+  
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet"> 
+</head>
+<body>
+  <!-- Navigation Bar -->
+  <div class="container navbar">
+    <nav>
+      <div class="logo"><h4><a href="#home">My Portfolio</a></h4></div>
+      <ul class="nav-links">   
+        <li><a href="#home">Home</a></li>      
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#about">About</a></li>
+      </ul>
+    </nav>  
+  </div>
+
+  <!-- Main Section -->
+  <a id="home"></a>
+  <div class="main-info-section">
+    <div class="text">
+      <h1>
+        <span class="letter">H</span>
+        <span class="letter">i</span>
+        <span class="letter">,&nbsp</span>
+        <span class="letter"> I</span>
+        <span class="letter">'</span>
+        <span class="letter">m</span>
+        <span class="name">Godsfavour</span>
+      </h1>
+      <p> 
+        I am a Google STEP (Student Training in Engineering Program) intern, student,
+        and aspiring software developer.  
+      </p>     
+    </div>
+
+    <div class="links">
+      <ul class="external-links">   
+        <li><a href="https://www.linkedin.com/in/godsfavour-simon-7b8838195/">LinkedIn</a></li>      
+        <li><a href="https://github.com/godsfavours">GitHub</a></li>
+      </ul> 
+    </div>
+  </div>  
+
+  <!-- Projects Section -->
+  <a id="projects"></a>
+  <div class="projects-section">
+    <h1 class="header">My Projects</h1>
+    <p class="msg">View some of my work</p>
+    <div class="projects-list">
+      <a href="#home">
+      <div class="project"> 
+        <h2>Portfolio Website</h2>
+        <p>My introductory project to HTML, css, and javascript!</p>
+        <img src="images/portfolio-website.png" alt="">
+     
+    </div>
+    </a>
+  </div>
+  </div>
+
+  <!-- About Section -->
+  <a id="about"></a>
+  <div class="about-section">
+    <h1 class="header">About Me</h1>
+    <p>
```

Comment:
I would suggested taking a closer look at indentation and line breaks and keeping them consistent throughout.  This section here is a good example:  if I'm reading correctly I think you have an extra `</p>` which I was able to catch because the indentations/line breaks didn't match.

Extra `</p>` I don't think would hurt as much, but once you get into mismatching `<div>` tags, that'll break the layout pretty bad and would be hard to debug as well.

I'm not too familiar with the Cloud Shell editor, but other tools like VS Code or Eclipse I believe has smart features that would format files for you, and highlight extra/missing tags.